pkgs/samples/lib/google_ai.dart

```diff
@@ -178,7 +201,12 @@ class _ChatWidgetState extends State<ChatWidget> {
                   isFromUser: content.role == 'user',
                 );
               },
-              itemCount: history.length,
+              itemCount: _chat.history.length,
+            )
+                : ListView(
+              children: const [
+                Text('No API key found. Please provide an API Key.'),
```

Comment:
Ah good point to add this, since I think the login(?) page currently allows empty submissions.

We don't have a back button, so maybe we should say to restart the sample? However, it's hopefully a rare mistake, so it might be fine to not provide special details or handling to avoid diverging too much from the original sample.